### PR TITLE
Add back to normal time fact as duration

### LINKED ISSUE #69: Back to normal time reported as datetime instead of duration
After a failed build is fixed, one of the facts displayed is "Back to normal time." Instead of showing up as duration, however, it is shown as a datetime.

For example, it will say "Back to normal time: 1969-12-31 19:02:16 EST" instead of "Back to normal time: 2 minutes, 16 seconds"

### PR DESCRIPTION
**Pull request by ctomp**
Fixes #69

### CODE REVIEW

File: src/main/java/jenkins/plugins/office365connector/model/Fact.java [hunk 1]
```diff
@@ -63,4 +64,18 @@ public void setValue(List<String> causesStrList) {
             this.value += cause + ". ";
         }
     }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (damianszczepanik) [inline src/main/java/jenkins/plugins/office365connector/model/Fact.java]: it is used anywhere?

Comment (ctomp) [reply src/main/java/jenkins/plugins/office365connector/model/Fact.java]: I used it in FactsBuilderTest when comparing the expected Fact to the actual Fact that was built.

Comment (damianszczepanik) [reply src/main/java/jenkins/plugins/office365connector/model/Fact.java]: then the question is: do we want to have this code only to satisfy test? IMO it is possible to rewrite test so those methods are not needed

Comment (ctomp) [reply src/main/java/jenkins/plugins/office365connector/model/Fact.java]: I agree, I'll go ahead and change the test so I don't have to implement equals() and hashCode().

**Pull request merged**